### DISCUSSION TITLE
Hi Tobexyz, I need to build issue 82 debug apk, right?

### DISCUSSION THREAD

**Discussion by Mahava** [tobexyz/yaacc-code#83] [Q&A]
I couldnt build it, it failed.   In fairness, Im a novice, so I might have set up the environment wrong.

**Comment by tobexyz** [comment]
Hi, yes you have to build the project, as described in the readme. After that you will get an debug APK which can be installed on your phone.
Alternatively you can build the project using the android studio and launch the app directly out of the ide on your phone. For this you have to activate the developer options on the phone and attach the phone to your computer for debugging. Hope this helps a bit :)
